### PR TITLE
Add gesture related clarification to docs and simple example.

### PR DESCRIPTION
**Pull request by ntoll**

### CODE REVIEW

**Comment by ntoll** [comment]
Any takers for this..? @dpgeorge..???

**Comment by dpgeorge** [comment]
I'll take a look.

File: docs/accelerometer.rst [hunk 1]
```diff
@@ -34,11 +34,18 @@ Functions
     Get the acceleration measurements in all axes at once, as a three-element
     tuple of integers ordered as X, Y, Z.
 
+.. note::
```
Comment (dpgeorge) [inline docs/accelerometer.rst]: Shouldn't this note go after the current_gesture() function's description?

**Comment by dpgeorge** [comment]
I tested the 8-ball example and it works :)

**Comment by ntoll** [comment]
OK... revised (you're right!).

**Comment by dpgeorge** [comment]
Please rebase :)

**Comment by dpgeorge** [comment]
hold on, i'll merge something else first

**Comment by dpgeorge** [comment]
@ntoll, rebasing time :)

**Comment by ntoll** [comment]
Rebased with master.

**Pull request merged**